code/codegen/src/main/kotlin/com/android/gradle/replicator/codegen/gradlegen/groovy/GradleKotlinBuildFileGenerator.kt

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2020 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.android.gradle.replicator.codegen.gradlegen.groovy
+
+import com.android.gradle.replicator.codegen.PrettyPrintStream
+import com.android.gradle.replicator.codegen.gradlegen.BuildFileGenerator
+
+/**
+ * Build file generator for plain Java project building with Gradle.
+ */
+class GradleKotlinBuildFileGenerator : BuildFileGenerator {
+    override fun generate(dependencies: List<String>, printer: PrettyPrintStream) {
+        printer.addBlock("plugins {")
```

Comment:
I have a better API for it in the generator but it might not be accessible here (especially if that moves to test code). We can improve later.